sage_patchbot/patchbot.py

```diff
@@ -633,10 +655,13 @@ def get_local_config(self):
 
         # plugin setup
         if not conf['plugin_only']:
-            active_plugins = conf['plugins']
-            if "docbuild" not in active_plugins:
-                # force building the doc, so that the tests can pass
-                conf['plugins'] = active_plugins + ["docbuild"]
+            global plugins_available
+            plugins = set(conf['plugins'])
+            plugins.update(conf["plugins_enabled"])
+            plugins.difference_update(conf["plugins_disabled"])
+            plugins.add("docbuild")   # docbuild is mandatory so that tests pass
+
+            conf["plugins"] = [p for p in plugins_available if p in plugins]
 
```

Comment:
il faudrait faire ca en dehors de la condition "if not conf['plugin_only']" qui doit seulement servir a activer "docbuild"